lib/utils/helpers/getComponent.tsx

```diff
@@ -18,6 +19,7 @@ export const GetTailwindInputComponent = (
         value={formData}
         type={"number"}
         onChange={handleInputChange}
+        onBlur={onBlur}
         description={description}
         errors={errors}
       />
```

Comment:
for this condition, use switch case is better and readability more.

Reply:
done